src/bci_build/package/389-ds/README.md.j2

```diff
@@ -0,0 +1,67 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+[389 Directory Server](https://www.port389.org/) is a highly usable, fully featured, reliable and secure LDAP server implementation.
+
+## Usage
+
+By default, the image launches 389 Directory Server with the same configuration that comes with the SUSE Linux Enterprise Server, with one exception, a pre-configured Name Service Switch (NSS) configuration file (`/etc/nsswitch.conf`) in included.
+
+```ShellSession
+$ podman run -it --rm -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Volumes
+
+The database is stored in the directory `/data`. A new empty database will be created on container startup unless an existing database is already present in `/data`.
+
+To mount a host directory as a volume for your database, run the following command:
+
+```ShellSession
+$ podman run -it --rm -v /my/own/datadir:/data:Z -p 3389:3389 -p 3636:3636 {{ image.pretty_reference }}
+```
+
+## Certificates
+
+By default, the container will use a self-signed CA certificate and a server certificate signed by that CA.
+
+A custom TLS certificate should be placed in `/data/tls/server.crt` and the key should be placed in and `/data/tls/server.key`.
+The CA certificates should be placed under `/data/tls/ca/` in separate files, i.e., `/data/tls/ca/ca1.crt` and `/data/tls/ca/ca2.crt`.
+
+## Environment variables
+
+### DS_ERRORLOG_LEVEL
+
+This optional environment variable can be used to set the log level for `ns-slapd` (default is `266354688`).
```

Comment:
The log level is `266354688`? I think I need an explanation of their log level logic :sweat_smile:

Reply:
Oh, it seems this is the reason:

> levels are additive, to enable several different kinds of logging, add the values of those levels together. 

Take from https://access.redhat.com/documentation/en-us/red_hat_directory_server/11/html/configuration_command_and_file_reference/logs-reference

Not even sure what to document here, it seems no 389 DS docs include information about log levels, only OpenLDAP has it.

Reply:
> Oh, it seems this is the reason:
> 
> > levels are additive, to enable several different kinds of logging, add the values of those levels together.
> 
> Take from https://access.redhat.com/documentation/en-us/red_hat_directory_server/11/html/configuration_command_and_file_reference/logs-reference
> 

But the linked page only mentions log levels up to 512. How do they end at these insanely large numbers?

If you can't find the info, then I'd rather leave it out to not confuse people even more.

Reply:
A few more are documented here https://www.openldap.org/doc/admin24/slapdconfig.html.

The full level list is in the code https://github.com/389ds/389-ds-base/blob/main/ldap/servers/slapd/proto-slap.h#L809 but is not documented.

It is still worth documenting the value, but I'm not sure where I should send users to, 389ds seems to completely ignore documenting these values.

Reply:
It does not hurt to ask https://github.com/389ds/389-ds-base/issues/6231.

Reply:
Would you be fine with parking this for e.g. a week and see how upstream responds? If they don't then we just merge this without a proper explanation

Reply:
I got feedback already, it covers most log levels. I can work around and document the levels on our side, using a markdown table (if supported in the registry).

I'm also fine waiting til next week to see if there are more comments on the upstream issue.

Reply:
Seems like you already found all the log level docs, lmk if you need me to explain any others.